e2e/browser/test-app/components/appContent/index.tsx

```diff
@@ -0,0 +1,136 @@
+//
+// Copyright 2022 Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import {
+  login,
+  logout,
+  handleIncomingRedirect,
+} from "@inrupt/solid-client-authn-browser";
+import type { ISessionInfo } from "@inrupt/solid-client-authn-browser";
+import {
+  TESTID_ERROR_MESSAGE,
+  TESTID_LOGIN_BUTTON,
+  TESTID_LOGOUT_BUTTON,
+  TESTID_OPENID_PROVIDER_INPUT,
+  TESTID_SESSION_STATUS,
+} from "@inrupt/internal-playwright-testids";
+import { useState, useEffect } from "react";
+import VerifiableCredential from "../verifiableCredentials";
+
+// This is the content of the file uploaded manually at SHARED_FILE_IRI.
+const DEFAULT_ISSUER = "https://login.inrupt.com/";
+const REDIRECT_URL = window.location.href;
+const APP_NAME = "Access Grants browser-based tests app";
+const AccessGrantContainer = ({
```

Comment:
Even if the VCs here happen to be access grants, I would be in favour of gearing the naming towards VC not to emphasis on the data model, but on its spec-compliant container.